book/files-modules-and-programs/README.md

```diff
@@ -896,17 +899,17 @@ val median : t -> median
 will lead to a compilation error.
 
 ```sh dir=examples/erroneous/freq-with-type-mismatch
-$ dune build freq.bc
+$ dune build freq.exe
 File "counter.ml", line 1:
 Error: The implementation counter.ml
-       does not match the interface .freq.eobjs/byte/counter.cmi:
+       does not match the interface .freq.eobjs/byte/dune__exe__Counter.cmi:
```

Comment:
This looks messed up.  What's the dune__exe nonsense showing up here?

Reply:
@avsm @NathanReb Do either of you know why bumping the Dune version would regress the error messages in this way?